image_width: 1280
image_height: 720

ros_rate: 25
camera_topic: "/anafi/image_raw"
